extensions/ql-vscode/src/cli.ts

```diff
@@ -509,12 +509,12 @@ export class CodeQLCliServer implements Disposable {
     testPaths: string[], workspaces: string[], options: TestRunOptions
   ): AsyncGenerator<TestCompleted, void, unknown> {
 
-    const subcommandArgs = [
+    const subcommandArgs = this.cliConfig.additionalTestArguments.toString().split(' ').concat([
```

Comment:
This will create empty arguments if there are duplicate spaces. Try using a regex instead. I think this will work better.

```suggestion
    const subcommandArgs = this.cliConfig.additionalTestArguments.toString().split(/\s+/).concat([
```